CONTRIBUTING.md

```diff
@@ -1,61 +1,686 @@
-# Contributing to this repository
+# Contributing to Pipelinit
 
-Thank you for your interest in contributing to pipelinit!
+Thanks for your interest in contributing to Pipelinit. If this is your first contribution, please read our code of conduct and the Pipelinit core concepts.
```

Comment:
Add a shortcut to the code of conduct (ex: `[code of conduct](CODE_OF_CONDUCT.md)`), add a shortcut to the "core concepts" too.

Reply:
Updated!